### ISSUE TITLE
Work with newer gulp-cli 2.0.0?

### ISSUE THREAD

**Issue by bdelaney** [nickdodd79/vscode-gulptasks#7]
Just installed in project with gulpfile and it locates that fine. But then hangs on 'loading gulp tasks ...'

**Comment by nickdodd79** [comment]
Hi @bdelaney,

Thanks for the heads up, I'll take a look and put a fix in.

Nick.

**Comment by bdelaney** [comment]
I did test against regular Gulp client in same project and the extension works fine.
I'm not sure it is an issue for the extension though. By reading some issues on the gulp-cli repo I found that the gulp-cli install is not supposed to need the old gulp package, and yet, it fails in a local project with this error if the old gulp package is not installed:
`Local gulp not found
Try running: npm install gulp`
It's odd because some gulp commands work, but others give this error.
So I npm install the old gulp client locally in the project...
Voila! Your extension works fine and loads Gulp tasks.

Very odd behavior by the new gulp-cli which does not indicate any dependency on old gulp package.

**Comment by nickdodd79** [comment]
Hi @bdelaney, 

I have just spent some time trying to understand why the issue was occurring and reached the same conclusion. It seems that there needs to a local gulp install for the tasks to be discovered. 

The extension was hanging as such, but swallowing an empty result which looked like nothing was happening. It executes 'gulp --tasks-simple' which was returning an empty string and not display a message correctly. I have changed this slightly so at least a message is displayed.

Unfortunately though I don't think I can make any changes to fix the gulp-cli you found, other than installing gulp local to the gulpfile.js project.

Nick.
